osu.Game/Overlays/Chat/ChannelControl/ControlItemText.cs

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+#nullable enable
+
+using osu.Framework.Allocation;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Sprites;
+using osu.Game.Online.Chat;
+
+namespace osu.Game.Overlays.Chat.ChannelControl
+{
+    public class ControlItemText : Container
```

Comment:
Would also argue there's no point in adding this class. It's 99% data binding from the parent just to update one string.